### PR TITLE
Drawing tiles

### LINKED ISSUE #4: Draw Tile using drawing methods instead of texture
Right now, Tiles are initialized with an image, but they should be drawn using the drawing methods in Pixi.js instead. This should make them more performant and extensible.

### PR DESCRIPTION
**Pull request by daltonclaybrook**
Resolves #4

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
   "semi": false,
-  "singleQuote": true
-}
\ No newline at end of file
+  "singleQuote": true,
+  "tabWidth": 2,
+  "printWidth": 120
```
Comment (daltonclaybrook) [inline .prettierrc:5]: The default is `80` and I feel like this is too little. Let me know if you disagree and I will switch it back.

File: src/tile-graphics.ts [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+  // MARK: - Helpers
+
+  private drawRoadSegments = (tile: TileModel, graphics: Graphics) => {
+    const drawCarRoad = this.curriedDrawCarRoad(tile, graphics)
+    const drawTrainRoad = this.curriedDrawTrainRoad(tile, graphics)
+
+    if (tile.top == RoadType.Car) {
+      drawCarRoad(RoadSide.Top)
+    } else if (tile.top == RoadType.Train) {
+      drawTrainRoad(RoadSide.Top)
```
Comment (drewwyatt) [inline src/tile-graphics.ts:55]: Train Road!

File: src/tile-model-factory.ts [hunk 3]
```diff
@@ -0,0 +1,136 @@
+import { TileModel, RoadType } from './tile-model'
+
+const makeTileModel = (id: number): TileModel => {
```
Comment (drewwyatt) [inline src/tile-model-factory.ts:3]: should/could `id` be an `enum`?

**Review by drewwyatt — APPROVED** [review]

File: src/tile-graphics.ts (hunk 2 above)
Comment (daltonclaybrook) [reply src/tile-graphics.ts:55]: lol. Definitely open to renaming if it's silly. Maybe something like `path`?

File: src/tile-model-factory.ts (hunk 3 above)
Comment (daltonclaybrook) [reply src/tile-model-factory.ts:3]: What would you name each case?

File: src/tile-graphics.ts (hunk 2 above)
Comment (drewwyatt) [reply src/tile-graphics.ts:55]: Nah. I think this makes total sense

File: src/tile-model-factory.ts (hunk 3 above)
Comment (drewwyatt) [reply src/tile-model-factory.ts:3]: true. I guess that would probably mean that we would need to name each tile

Comment (daltonclaybrook) [reply src/tile-model-factory.ts:3]: Happy to make this an `enum` if we can think of a good naming schema.

**Pull request merged**